package.json

```diff
@@ -128,6 +128,20 @@
         ],
         "visible": true
       },
+      {
+        "permissionName": "settings.data-export.view",
+        "displayName": "Settings (Data export): Can view",
+        "subPermissions": [
+          "data-export.mapping-profiles.item.get",
+          "data-export.job-executions.collection.get",
+          "data-export.job-profiles.collection.get",
+          "data-export.job-profiles.item.get",
+          "data-export.transformation-fields.collection.get",
+          "data-export.logs.collection.get",
+          "data-export.mapping-profiles.collection.get"
+        ],
+        "visible": false
```

Comment:
I think visible should be true, as it is intended to be assigned to a user (from permissions modal) and be able to view (and not edit) the settings pages.

Reply:
Ye, agree. Done!